### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by octonato**
* Akka 2.5.21
 * Play 2.7.0
 * Lagom 1.5.0-RC2
 * Akka gRPC 0.5.0

### CODE REVIEW

File: akka-grpc-play-quickstart-scala/project/plugins.sbt [hunk 1]
```diff
@@ -10,5 +10,5 @@ addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.0-M3")
 addSbtPlugin("com.typesafe.sbt" % "sbt-play-enhancer" % "1.2.2")
 
 // Akka GRPC
-addSbtPlugin("com.lightbend.akka.grpc" %% "sbt-akka-grpc" % "0.4.1")
+addSbtPlugin("com.lightbend.akka.grpc" %% "sbt-akka-grpc" % "0.5.0")
```
Comment (ignasi35) [inline akka-grpc-play-quickstart-scala/project/plugins.sbt:13]: Does this work? I thought `addSbtPlugin("com.lightbend.akka.grpc" %% "sbt-akka-grpc" % "0.5.0")` included an issue on the play generator.

File: akka-grpc-lagom-quickstart-scala/project/plugins.sbt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // The Lagom plugin
-addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.5.0-RC1")
+addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.5.0-RC2")
 // Akka GRPC
-addSbtPlugin("com.lightbend.akka.grpc" %% "sbt-akka-grpc" % "0.4.2")
+addSbtPlugin("com.lightbend.akka.grpc" %% "sbt-akka-grpc" % "0.5.0")
```
Comment (ignasi35) [inline akka-grpc-lagom-quickstart-scala/project/plugins.sbt:4]: same thing. Does this work?

File: akka-grpc-play-quickstart-scala/project/plugins.sbt (hunk 1 above)
Comment (octonato) [reply akka-grpc-play-quickstart-scala/project/plugins.sbt:13]: I have done so before and it works. Unless there is a new bug introduced in the meantime.

Comment (ignasi35) [reply akka-grpc-play-quickstart-scala/project/plugins.sbt:13]: I forgot, the bug is only on `javadsl` (IIRC in the generated Guice `Module`)

**Review by ignasi35 — APPROVED** [review]

**Comment by octonato** [comment]
Out-of-date.

**Pull request closed without merging**